Lib/idlelib/config.py

```diff
@@ -192,7 +193,7 @@ def GetUserCfgDir(self):
                     except OSError:
                         pass
                 userDir = '~'
-        if userDir == "~": # still no path to home!
+        else: # still no path to home!
```

Comment:
This changes behavior because line 194 may change `userDir`.

Reply:
Reverted change